stock-filters/ChirpVillage/Builder.py

```diff
@@ -41,15 +45,27 @@ def construct_floor_and_flat_roof(level, coords, biome, height):
             utilityFunctions.setBlock(
                 level, floor_block, x, miny, z)
 
+
 def construct_pointed_roof(level, coords, biome, height):
     minx, minz, miny, maxx, maxz = get_coords(coords)
     roof_block = BlockUtils.get_roof_block(biome)
-    for i in range(-1, maxx-minx/4):
+    for i in range(-1, abs(maxx-minx)/4):
```

Comment:
this a bug fix?

Reply:
Yeah. In some situations the maxx-minx would be negative.